frontend/src/utils/hooks/useCreateChartTitle.tsx

```diff
@@ -0,0 +1,35 @@
+import { useMediaQuery, createTheme } from "@material-ui/core";
+import { MetricConfig } from "../../data/config/MetricConfig";
+
+const theme = createTheme({
+  breakpoints: {
+    values: {
+      xs: 0,
+      sm: 800,
+      md: 900,
+      lg: 1500,
+      xl: 1850,
```

Comment:
i dont think these are the material breakpoints we use elsewhere. These are what i had found and added to the variable files a while ago : https://github.com/SatcherInstitute/health-equity-tracker/blob/main/frontend/src/styles/variables.module.scss#L105-L109

Reply:
Maybe these were intentional choices for px values to make the layout work? if so that's totally fine but we should indicate that these breakpoints are different than the others we use for the rest of the layour responsiveness

Reply:
The default breakpoints don't do well for mocking the text wrapping of the chart titles. I'll add some comments for them.